meta/main..yml

```diff
@@ -0,0 +1,51 @@
+galaxy_info:
+  author: Hector Sanjuan
+  description: Ansible role to install and run go-ipfs and IPFS Cluster
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Choose a valid license ID from https://spdx.org - some suggested licenses:
+  # - BSD-3-Clause (default)
+  # - MIT
+  # - GPL-2.0-or-later
+  # - GPL-3.0-only
+  # - Apache-2.0
+  # - CC-BY-4.0
+  license: license (GPL-2.0-or-later, MIT, etc)
```

Comment:
License is MIT.

Reply:
fixed in https://github.com/hsanjuan/ansible-ipfs-cluster/pull/10/commits/c05ab6d354b2b5eddbbc6031c735b01999195a33